### ISSUE TITLE
ugly ?

### ISSUE THREAD

**Issue by LesChats** [Reaster0/ft_transcendence#130]
<img width="684" alt="Screen Shot 2022-06-25 at 5 50 58 PM" src="https://user-images.githubusercontent.com/51509223/175781183-718f365d-ef8b-463a-8762-b943f521f0c0.png">

**Comment by Reaster0** [comment]
yeah totally agree with you, it's been a while since i said "i will make it prettier when i have an idea of how i will do it" i still don't have any ideas so if you have you can make it prettier :')
